src/Features/App/Navbar/Navbar.tsx

```diff
@@ -151,13 +151,14 @@ export default function NavbarContainer({ handleOnTutorialClick, navigationData,
     requirePlatformConsent: isStandaAloneMode ? false : true,
     renderLogo: true,
   };
-
+  const platformName = navigationData?.platform?.platformName ?? "Boomerang";
   const isAtLeastOperator = userData.type === UserType.Admin || userData.type === UserType.Operator;
   return (
     <>
-      <Helmet>
-        <title>{`Flow | ${navigationData.platform?.platformName ?? "Boomerang"}`}</title>
-      </Helmet>
+      <Helmet
+        defaultTitle={`Boomerang Flow`}
+        titleTemplate={`%s - Boomerang Flow - ${platformName}`}
```

Comment:
Just a comment: because of this format we are showing 4 dash separated items in the helmet in Execution and Editor.

Reply:
@LucasGrimauth I ended up changing things in CICD to have four dashes in some situations. Not ideal but I think its fine. (I know I said otherwise before but I couldn't come up with a good solution)